src/scripts/main.js

```diff
@@ -1,3 +1,286 @@
 'use strict';
 
-// write code here
+const titles = document.querySelectorAll('table th');
+const rows = Array.from(document.querySelectorAll('tbody tr'));
+let click = 0;
+
+const form = document.createElement('form');
+const label1 = document.createElement('label');
+const label2 = document.createElement('label');
```

Comment:
Add more descriptive name for variables. Fix it everywhere For example: 
```suggestion
const labelName = document.createElement('label');
const labelPosition = document.createElement('label');
```